lua/lean/infoview.lua

```diff
@@ -36,6 +36,7 @@ local options = {
     indicators = "auto",
     lean3 = { show_filter = true, mouse_events = false },
     show_processing = true,
+    show_no_info = false,
```

Comment:
```suggestion
    show_no_info_message = false,
```

Bit clearer to me perhaps what this does, even if it's longer.